process/graduation_criteria.md

```diff
@@ -14,6 +14,7 @@ To be accepted in the sandbox a project must
 * Adopt the CNCF [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
 * Adhere to CNCF [IP Policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy) (including trademark transferred)
 * List their sandbox status prominently on website/readme
+* Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md or MAINTAINERS.md file showing the current and emeritus committers. See https://contribute.cncf.io/maintainers/governance/ for more info.
```

Comment:
```suggestion
* Explicitly define a project governance and maintainer selection process. The maintainer process should cover the full maintainer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md or MAINTAINERS.md file showing the current and emeritus maintainer. See https://contribute.cncf.io/maintainers/governance/ for more info.
```

Please stop using the word "committer".  It is misleading and confusing to projects.

Reply:
Yeah, I'm trying to scrub it everywhere I see it.  And we should amend the charter to remove it.

Reply:
I defer to your experience here Josh, but I don't understand the motivation - could you share more please? Is this a personal request, or to align with some broader industry change I was not aware of?

Reply:
Because, if you use git, *every single contributor is a committer*.  Look at the "committer" stats in DevStats, the official metrics for CNCF projects.  They include *everyone with a merged PR*.

So, for any potential contributor who's been in OSS less than 10 years, using the word "committer" to refer to project leadership is profoundly confusing.  It's the equivalent of describing a retail business, and using the word "associates" as being the same as "management". So, yes, it's an industry change.  Every time a TOC member says "committer" they sound 60 years old.

In addition to this, many/most CNCF projects have people who review and approve PRs who are not otherwise project leadership -- and people who are project leadership who don't do much approving PRs.   

And, further, all of the documentation from CNCF staff, including all of the mailing lists, etc., refer to "maintainers".  Having a single, clear term, which doesn't mean two radically different things is much better for everyone.

Reply:
> Every time a TOC member says "committer" they sound 60 years old.

This ageist comment has no place in a CNCF discussion.

It feels to me like you've made this point about changing "committer" to "maintainer" over and over, but there's no evidence of a groundswell of support. I will keep making the point that we do not define governance, so projects can use whatever terminology suits them best. The language of the Charter uses the word "committer", and changing this needs a Governing Board discussion, so unless someone from the TOC and/or GB want to take it up, going round in circles on this doesn't seem like a great use of time.

Reply:
I echo Liz's sentiment about the 60 year old comment here.

The comment could imply that there is a requirement for a TOC member to be under 60 years old.